src/ripple/ledger/impl/View.cpp

```diff
@@ -391,7 +391,13 @@ transferRate (ReadView const& view,
     auto const sle = view.read(keylet::account(issuer));
 
     if (sle && sle->isFieldPresent (sfTransferRate))
-        return Rate{ sle->getFieldU32 (sfTransferRate) };
+    {
+        auto rate = (*sle)[sfTransferRate];
+        auto maxRate = 2 * QUALITY_ONE;
```

Comment:
std::min requires both numbers to be the same type - since rate is a uint32_t, and 2 * QUALITY_ONE is a signed int, we have to make them uniform. I ended up removing the maxRate instantiation and casting it as an unsigned int.

However, I'd vote for keeping in maxRate because it makes the code a bit more readable.